### DISCUSSION TITLE
change field to another type

### DISCUSSION THREAD

**Discussion by Enhed** [directus/directus#6506] [Archive]
from #6504 

i change it in database, how to change into code and create PR? i create fork directus but dunno where is it info and how to check after changes

or maybe anyone can help with change that? its very important for my project
ps with bigint sizes not shows as MB, just source number, i guess need change and vue module

it is real for that update will be in main project?

**Comment by rijkvanzanten** [comment]
You'll have to write a migration (https://github.com/directus/directus/tree/main/api/src/database/migrations) to make sure the field is updated correctly for new and existing installations

**Reply by Enhed** [reply]
https://github.com/Enhed/directus/commit/85eb8a09fcbb8cdf9a1f5e2e3ac0bf3d39f114e6
im not sure that correctly, can u check?

**Reply by Enhed** [reply]
and what next i need to do for PR?

**Reply by rijkvanzanten** [reply]
That looks good to me! 

> and what next i need to do for PR?

This is a pretty good step by step overview of how to do contributions on GitHub: https://github.com/firstcontributions/first-contributions#submit-your-changes-for-review 🙂
